server/src/main/resources/file.conf

```diff
@@ -48,3 +38,31 @@ client {
     retry.times = 30
   }
 }
+
+## transaction log store
+store {
+  ## store mode: file、db
+  mode = "file"
+
+  ## file store
+  file {
+    dir = "root.data"
+  }
+
+  ## database store
+  db {
+    driver_class = "x"
+    url = "x"
+    user = "x"
+    password = "x"
+  }
+
+  # branch session size , if exceeded first try compress lockkey, still exceeded throws exceptions
+  max-branch-session-size = 16384
+  # globe session size , if exceeded throws exceptions
+  max-global-session-size = 512
+  # file buffer size , if exceeded allocate new buffer
+  file-write-buffer-cache-size = 16384
```

Comment:
should move to under the file node？also modify ConfigurationKey in the code.
![image](https://user-images.githubusercontent.com/8758457/56107620-eccc3c00-5f79-11e9-95c5-35152c282216.png)

Reply:
> should move to under the file node？also modify ConfigurationKey in the code.
> ![image](https://user-images.githubusercontent.com/8758457/56107620-eccc3c00-5f79-11e9-95c5-35152c282216.png)

done.